[gSoap] Update to 2.8.102 and re-enable x64 Builds

xgcssch opened the pull request:
This PR updates the port to upstream version 2.8.102
Furthermore it re-enables the builds for x64, which were erroneously disabled

xgcssch committed:
Update to 2.8.102 and reenable x64 Build

xgcssch committed:
Remove surplus empty directory during install

BillyONeal commented:
/azp run

strega-nil commented on ports/gsoap/CONTROL:
```diff
@@ -1,6 +1,6 @@
 Description: The gSOAP toolkit is a C and C++ software development toolkit for SOAP and REST XML Web services and generic C/C++ XML data bindings.
-Supports: x86 & windows
\ No newline at end of file
+Supports: x86 & x64 & windows
```
This is broken (and for some reason we don't catch it in CI; cc @ras0219-msft). This should be `(x86 | x64) & windows`

xgcssch replied:
Performed the supposed change.

xgcssch committed:
Corrected the "Supports" entry in CONTROL

Should have read the documentation for "Supports" to see that this is a boolean expression

strega-nil commented:
Alright, awesome! Thanks @xgcssch :)
